README.md

```diff
@@ -165,12 +165,13 @@ magazines                                 downloads specific magazine issue as a
                       [url]               shows the homepage URL of magazine
 resolution <cea|dmt [modes]>              sets the screen resolution
 system [cpu|ram|disk|volt|temperature]    display real system informations
-message gitter <apitoken|authorize>       sets api/channel info in config file and sends/recieves messages in gitter
-               <send|show|read|mark>
-               <channels>
-        slack <apitoken|channels>         sets api/channel info in config file and sends/recieves messages in slack
-              <send|show|read|mark>
-        discord <apitoken|authorize>      sets api/channel info in config file
+message                                   sends message to chat service
+   <gitter>  <apitoken|authorize>         sets api/channel info in config file and sends/recieves messages in gitter
+             <send|show|read|mark>
+             <channels>
+   <slack>   <apitoken|channels>          sets api/channel info in config file and sends/recieves messages in slack
+             <send|show|read|mark>
+   <discord> <apitoken|authorize>         sets api/channel info in config file
```

Comment:
every change in README.md should also be in help.sh

Reply:
The new format for the subcommands should also be included in the message.sh file at the end (message_help function).

Reply:
> 
> 
> The new format for the subcommands should also be included in the message.sh file at the end (message_help function).

Yah....
I should have just made a single issue to fix both (see #2049)